repo-update.sh

```diff
@@ -24,6 +24,7 @@ function update {
   echo $editedFiles
   git checkout master
   git pull --rebase
+  git remote update --prune
```

Comment:
And here you can then simply add an if condition with `prune_remote` set to false by default:

```
if [ "$prune_remote" = true ] ; then
    git remote update --prune
if
```